### PR TITLE
{lang}[GCCcore/13.2.0] Python-bundle-PyPI v2023.10

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)
- [x] requires #19099

### CODE REVIEW

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
alvis-c1 - Linux Rocky Linux 8.8, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/Micket/bd962167f27f514e873ccd15d95f7e04 for a full test report.

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
vera-c1 - Linux Rocky Linux 8.8, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/Micket/5aae2750cd8d330b6ad9e73004a8a393 for a full test report.

**Comment by Micket** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/p/Python-bundle-PyPI/Python-bundle-PyPI-2023.10-GCCcore-13.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,480 @@
...
+        'checksums': ['3fef4f844d2290ee0ba57addcec17eec9e3df73f10a2748485dfd6a3a188cc0f'],
+    }),
+    ('intreehooks', '1.0', {
+        'checksums': ['87e600d3b16b97ed219c078681260639e77ef5a17c0e0dbdd5a302f99b4e34e1'],
+    }),
+    ('pylev', '1.4.0', {
+        'checksums': ['9e77e941042ad3a4cc305dcdf2b2dec1aec2fbe3dd9015d2698ad02b173006d1'],
+    }),
+    ('pastel', '0.2.1', {
+        'source_tmpl': '%(name)s-%(version)s-py2.py3-none-any.whl',
```
Comment (bedroge) [inline easybuild/easyconfigs/p/Python-bundle-PyPI/Python-bundle-PyPI-2023.10-GCCcore-13.2.0.eb:335]: Just wondering if we shouldn't stick to using `SOURCE_WHL` / `SOURCE_PY3_WHL` for these wheel sources as in https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/p/Python-bundle-PyPI/Python-bundle-PyPI-2023.06-GCCcore-12.3.0.eb?

Comment (Micket) [reply easybuild/easyconfigs/p/Python-bundle-PyPI/Python-bundle-PyPI-2023.10-GCCcore-13.2.0.eb:335]: we should, i just missed it (and inject checksums replaces these with the expanded template)

**Comment by bedroge** [comment]
@boegelbot please test @ jsc-zen2

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 6 out of 6 (2 easyconfigs in total)
interactive2 - Linux Rocky Linux 8.8, x86_64, AMD EPYC-Milan Processor (zen2), Python 3.6.8
See https://gist.github.com/bedroge/915a8892d930fef404320f7952796bf6 for a full test report.

**Review by bedroge — APPROVED** [review]
Lgtm

**Comment by bedroge** [comment]
Going in, thanks @Micket!

**Comment by jose-d** [comment]
@Micket  `Pygments` are twice there in Python-bundle-PyPI-2023.10-GCCcore-13.2.0.eb, but indeed that's not very critical..

**Pull request merged**